setup_env.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash -e
+# Ensure we use Go version 1.11+
+read major minor patch <<< $(go version | sed 's/go version go\([0-9]*\)\.\([0-9]*\)\.\(.*\) .*/\1 \2 \3/')
```

Comment:
is sed found in a clean ubuntu setup?

Reply:
Looks that way.

The [UNIX specification](https://en.wikipedia.org/wiki/List_of_Unix_commands) declares that it's mandatory:
![image](https://user-images.githubusercontent.com/776358/51673530-e2cc1a00-1fd6-11e9-89b0-5aa12b54b048.png)

A random [dude on AskUbuntu](https://askubuntu.com/a/957815) puts it nicely:

> `sed` is a utility that programs are generally allowed to assume exists and to rely on themselves. As you noticed, the scripts that run when installing, removing, or upgrading software can use `sed`.